release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -1300,6 +1300,12 @@ The Amazon Web Services (AWS) Elastic File System (EFS) Technology Preview featu
 
 * Technology preview badges were displayed on the OpenShift Pipelines workflows in the web console, though OpenShift Pipelines was GA. The Technology preview badges are now removed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1945153[*BZ#1945153*])
 
+* Previously, pipelines created in the Git import flow for private repositories failed to run. This happened because the pipeline `ServiceAccount` did not use secrets created by the Git import flow for private Git repositories. With this update, you can add a secret name to the annotations of the pipeline `ServiceAccount` object, and add pipeline-specific annotations to the provided secret. As a result, pipelines for private Git repositories run successfully. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1970485[BZ#1970485])
```

Comment:
Nit pick: Maybe instead of `previously` you could say something like, `Before BLANK RELEASE, the pipelines created in the Git import flow for private repositories failed to run.` 

I think this rephrase could be more specific and help with translation. However, it's up to you as `previously` works fine here :)

Reply:
Change the second sentence to: ... because the pipeline `ServiceAccount` object did not use secrets

Reply:
Done